src/istgt_iscsi.h

```diff
@@ -35,63 +35,63 @@
 #include "istgt_lu.h"
 #include "istgt_queue.h"
 #include <sys/uio.h>
-#define ISCSI_BHS_LEN 48
-#define ISCSI_ALIGNMENT 4
+#define	ISCSI_BHS_LEN 48
+#define	ISCSI_ALIGNMENT 4
 /* support version - RFC3720(10.12.4) */
-#define ISCSI_VERSION 0x00
+#define	ISCSI_VERSION 0x00
 
-#define ISCSI_ALIGN(SIZE) \
+#define	ISCSI_ALIGN(SIZE) \
 	(((SIZE) + (ISCSI_ALIGNMENT - 1)) & ~(ISCSI_ALIGNMENT - 1))
 
-#define ISCSI_TEXT_MAX_KEY_LEN 64
+#define	ISCSI_TEXT_MAX_KEY_LEN 64
 /* for authentication key (non encoded 1024bytes) RFC3720(5.1/11.1.4) */
-#define ISCSI_TEXT_MAX_VAL_LEN 8192
-
-#define SESS_MTX_LOCK(CONN) \
-	do {								\
-		int _rc_;						\
-		if ((_rc_ = pthread_mutex_lock(&(CONN)->sess->mutex)) != 0) {	\
-			ISTGT_ERRLOG("sess lock error:%d", _rc_);		\
-			pthread_exit(NULL);				\
-		}							\
+#define	ISCSI_TEXT_MAX_VAL_LEN 8192
+
+#define	SESS_MTX_LOCK(CONN) \
+	do {	\
+		int _rc_;	\
+		if ((_rc_ = pthread_mutex_lock(&(CONN)->sess->mutex)) != 0) {\
+			ISTGT_ERRLOG("sess lock error:%d", _rc_);	\
+			pthread_exit(NULL);	\
+		}	\
 	} while (0)
-#define SESS_MTX_UNLOCK(CONN) \
-	do {								\
-		int _rc_;						\
-		if ((_rc_ = pthread_mutex_unlock(&(CONN)->sess->mutex)) != 0) {	\
-			ISTGT_ERRLOG("sess unlock error:%d", _rc_);		\
-			pthread_exit(NULL);				\
-		}							\
+#define	SESS_MTX_UNLOCK(CONN) \
+	do {	\
+		int _rc_;	\
+		if ((_rc_ = pthread_mutex_unlock(&(CONN)->sess->mutex)) != 0) {\
+			ISTGT_ERRLOG("sess unlock error:%d", _rc_);	\
+			pthread_exit(NULL);	\
+		}	\
 	} while (0)
 
 typedef enum {
 	/* Initiator opcodes */
-	ISCSI_OP_NOPOUT         = 0x00,
-	ISCSI_OP_SCSI           = 0x01,
-	ISCSI_OP_TASK           = 0x02,
-	ISCSI_OP_LOGIN          = 0x03,
-	ISCSI_OP_TEXT           = 0x04,
-	ISCSI_OP_SCSI_DATAOUT   = 0x05,
-	ISCSI_OP_LOGOUT         = 0x06,
-	ISCSI_OP_SNACK          = 0x10,
-	ISCSI_OP_VENDOR_1C      = 0x1c,
-	ISCSI_OP_VENDOR_1D      = 0x1d,
-	ISCSI_OP_VENDOR_1E      = 0x1e,
+	ISCSI_OP_NOPOUT			= 0x00,
+	ISCSI_OP_SCSI			= 0x01,
+	ISCSI_OP_TASK			= 0x02,
+	ISCSI_OP_LOGIN			= 0x03,
+	ISCSI_OP_TEXT			= 0x04,
+	ISCSI_OP_SCSI_DATAOUT	= 0x05,
+	ISCSI_OP_LOGOUT			= 0x06,
+	ISCSI_OP_SNACK			= 0x10,
+	ISCSI_OP_VENDOR_1C		= 0x1c,
+	ISCSI_OP_VENDOR_1D		= 0x1d,
+	ISCSI_OP_VENDOR_1E		= 0x1e,
 
 	/* Target opcodes */
-	ISCSI_OP_NOPIN          = 0x20,
-	ISCSI_OP_SCSI_RSP       = 0x21,
-	ISCSI_OP_TASK_RSP       = 0x22,
-	ISCSI_OP_LOGIN_RSP      = 0x23,
-	ISCSI_OP_TEXT_RSP       = 0x24,
-	ISCSI_OP_SCSI_DATAIN    = 0x25,
-	ISCSI_OP_LOGOUT_RSP     = 0x26,
-	ISCSI_OP_R2T            = 0x31,
-	ISCSI_OP_ASYNC          = 0x32,
-	ISCSI_OP_VENDOR_3C      = 0x3c,
-	ISCSI_OP_VENDOR_3D      = 0x3d,
-	ISCSI_OP_VENDOR_3E      = 0x3e,
-	ISCSI_OP_REJECT         = 0x3f,
+	ISCSI_OP_NOPIN			= 0x20,
+	ISCSI_OP_SCSI_RSP		= 0x21,
+	ISCSI_OP_TASK_RSP		= 0x22,
+	ISCSI_OP_LOGIN_RSP		= 0x23,
+	ISCSI_OP_TEXT_RSP		= 0x24,
+	ISCSI_OP_SCSI_DATAIN	= 0x25,
```

Comment:
Can you please align this line with the others.

Reply:
Hi @payes , this is a weird issue, because if I clone the repo and open it in my editor (VS Code), it looks aligned to me. Check [here](https://imgur.com/a/HTn6gtV). Can you confirm? Because if I try to change the tab at this line, then somehow it's not working as expected.

Reply:
Do take a look at the recent commits I made to my fork trying to resolve this.

Reply:
The changes are done. Just letting you know that even though it now looks aligned on GitHub, it looks like [this](https://imgur.com/a/sKED4Il) in my editor (VS Code).